tests/Unit/App/Action/Backend/Order/UpdateOrderStatusActionTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/** @noinspection PhpUnhandledExceptionInspection,StaticClosureCanBeUsedInspection */
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Pdk\App\Action\Backend\Order;
+
+use MyParcelNL\Pdk\App\Api\Backend\PdkBackendActions;
+use MyParcelNL\Pdk\App\Order\Collection\PdkOrderCollection;
+use MyParcelNL\Pdk\App\Order\Model\PdkOrder;
+use MyParcelNL\Pdk\Base\Support\Arr;
+use MyParcelNL\Pdk\Facade\Actions;
+use MyParcelNL\Pdk\Facade\Notifications;
+use MyParcelNL\Pdk\Notification\Model\Notification;
+use MyParcelNL\Pdk\Tests\Uses\UsesApiMock;
+use MyParcelNL\Pdk\Tests\Uses\UsesMockPdkInstance;
+use MyParcelNL\Pdk\Tests\Uses\UsesNotificationsMock;
+use MyParcelNL\Pdk\Tests\Uses\UsesSettingsMock;
+use Symfony\Component\HttpFoundation\Response;
+use function MyParcelNL\Pdk\Tests\factory;
+use function MyParcelNL\Pdk\Tests\usesShared;
+
+usesShared(new UsesMockPdkInstance(), new UsesApiMock(), new UsesNotificationsMock(), new UsesSettingsMock());
+
+it('updates order status', function ($status) {
+    $orders = factory(PdkOrderCollection::class)
```

Comment:
```suggestion
    $orders = factory(PdkOrderCollection::class, 1)
```

dan kan die 

```php
        ->push(
            factory(PdkOrder::class)
        )
``` 

weg